packages/docs/src/routes/docs/concepts/resumable/index.mdx

```diff
@@ -48,6 +52,7 @@ Existing frameworks solve the event listener by downloading the components and e
 3. Requires the event handler code to be downloaded eagerly (to be attached).
 
 The above approach does not scale. As the application becomes more complicated, the amount of code needed to download eagerly and execute grows proportionally with the size of the application. This negatively impacts the application startup performance and hence the user experience.
+[//]: # (This is why frameworks introduced differents levels of lazy loading chunks of application js with different kind of granularity, wether being SPA or SSR.)
```

Comment:
yes, but this is a manual optimization devs need to do.. plus it is limited to routes and not closures...